### PR TITLE
refactor: update jsonrpc endpoint from *RecursiveFindContent to *GetContent

### PR DESCRIPTION
**Pull request by KolbyML**
### What was wrong?
updates our codebase to match spec https://github.com/ethereum/portal-network-specs/pull/344
### How was it fixed?

by renaming `portal_*RecursiveFindContent` to `portal_*GetContent`

### CODE REVIEW

File: ethportal-api/src/beacon.rs
Comment (morph-dev) [inline ethportal-api/src/beacon.rs:80]: I think it would be good to clarify that it will first check local storage.
Also in other places.

File: ethportal-api/src/types/portal.rs
Comment (morph-dev) [inline ethportal-api/src/types/portal.rs:20]: This object doesn't seem to be used at all.

File: ethportal-api/src/types/portal.rs
Comment (morph-dev) [inline ethportal-api/src/types/portal.rs:56]: While it's true that this is the response type, RecursiveFindContent would only return `Content` variant (this is also according to spec). Should we create separate type for it?

I also believe that `ConnectionId` variant is never used (not even for `FindContent`). Should we remove it?

While it's not strictly related to this pr (renaming endpoint), I think that this is something that we should improve. If you feel like it, maybe do it in this PR, otherwise create an issue for it.

File: ethportal-peertest/src/scenarios/gossip.rs
Comment (morph-dev) [inline ethportal-peertest/src/scenarios/gossip.rs:299]: nit: Unrelated to this pr, but this comment is wrong. Or at least, it looks like it was correct, but than line 301 was  commented out and line 302 is added. Would be nice to fix it if you know what is happening here.

**Review by morph-dev — APPROVED** [review]
:ship: 

I general it looks good to me. Couple of comments that maybe don't even belong to this PR, but I wanted to start discussion (if they are small enough, I wouldn't be against including them into this PR).

Since this is causing some API changes, please wait for at least one more approval.

File: ethportal-api/src/types/portal.rs
Comment (KolbyML) [reply ethportal-api/src/types/portal.rs:56]: I will make a follow up PR to resolve this

File: ethportal-peertest/src/scenarios/gossip.rs
Comment (KolbyML) [reply ethportal-peertest/src/scenarios/gossip.rs:299]: I removed it, that seems like an artifact left when writing the test

Comment (morph-dev) [reply ethportal-peertest/src/scenarios/gossip.rs:299]: The comment should still be updated to

File: ethportal-api/src/beacon.rs
Comment (morph-dev) [inline ethportal-api/src/beacon.rs:85]: nit: I think you forgot to update this doc (you did it in other files)

**Review by morph-dev — APPROVED** [review]

**Review by njgheorghita — APPROVED** [review]
:shipit:

**Pull request merged**
